src/Account.java

```diff
@@ -0,0 +1,64 @@
+public class Account {
+    private int number;
+    private int balance;
+    private String name;
+    private String email;
+    private int phone;
+
+    public int getNumber() {
```

Comment:
Nice. You can use Lombok plugin to generate getters and setters next time. All you need to to is to add:
@Getter
@Setter
public class Account {

at the top of the class declaration. Very usefull, we us it in our company. Unfortunately you need to download the plugin in your Intelij(it's free)

Reply:
I have already installed Lombok